### PR TITLE
Mesh only info

### PR DESCRIPTION
**Pull request by friedmud**
Print some info out when using `--mesh-only`

### CODE REVIEW

File: test/tests/mesh/mesh_only/tests [hunk 1]
```diff
@@ -8,6 +8,15 @@
     method = 'opt oprof'
   [../]
 
+  [./mesh_info]
+    type = 'RunApp'
+    input = 'mesh_only.i'
+    cli_args = '--mesh-only info_test.e'
+    recover = false
+    method = 'opt oprof'
+    expect_out = 'Mesh Information'
```
Comment (andrsd) [inline test/tests/mesh/mesh_only/tests]: This is missing `design`, `issue` and `requirement` parameters.  I checked and there is no `MeshOnlyAction.md`, so I assuming we need one?

**Review by andrsd — DISMISSED** [review]
Missing SQA

**Comment by andrsd** [comment]
I ran your code and I have a question: What do you think about a CLI flag `--print-mesh-info`? But, I am not a user of `--mesh-only`, so your change might be still ok.

**Comment by friedmud** [comment]
I don't know why you would ever want to not print it.

**Comment by andrsd** [comment]
I was thinking that `--mesh-only` would run the input file and produce the mesh and `--print-mesh-info` would read a mesh file, print info about it and quit, so one would do something like `./my-app --print-mesh-info <mesh-file>`

**Comment by friedmud** [comment]
What this does is prints mesh info while writing out the mesh - it's basic
information about what's going on.  It makes sense.  We seriously don't
need to spend time debating this.

Feel free to put up a PR for `--print-mesh-info` if you think it's useful -
it's tangential to this.

**Review by permcody — APPROVED** [review]

**Pull request merged**